### ISSUE TITLE
How to uninstall atom in osx

### ISSUE THREAD

**Issue by kimown** [atom/atom#15687]
Hi, I want to uninstall atom in mac,  I installed it from official site https://atom.io/, but I can't find a way to completely remove it, looks like it is a malicious software.
I have tried many ways ,like 
https://discuss.atom.io/t/how-to-completely-uninstall-atom-for-mac/9084

I know it's not a good place to answer this question, but I think it's better in doc atom should mention how to uninstall it, it has wasted others a lot of time.

**Comment by thomasjo** [comment]
Follow the steps provided the in the Discuss thread you have linked. The following looks like a complete removal;
```
rm -rf "/Applications/Atom.app"
rm -rf "/usr/local/bin/atom"
rm -rf "/usr/local/bin/apm"
rm -rf "~/.atom"
rm -rf "~/Library/Application Support/com.github.atom.ShipIt"
rm -rf "~/Library/Application Support/Atom"
rm -rf "~/Library/Caches/com.github.atom"
rm -rf "~/Library/Caches/Atom"
rm -rf "~/Library/Preferences/com.github.atom.plist"
rm -rf "~/Library/Saved Application State/com.github.atom.savedState"
```
_Based on https://discuss.atom.io/t/how-to-completely-uninstall-atom-for-mac/9084/34._

**Comment by kimown** [comment]
@thomasjo 
Thanks for your reply, I have tried these commands,
![image](https://user-images.githubusercontent.com/7932380/30578314-c44d4f1e-9d45-11e7-8f50-885e6fbd3126.png)

then I click a file,
![untitledremove](https://user-images.githubusercontent.com/7932380/30578466-a39ede6c-9d46-11e7-8707-83ad19a133bd.gif)

**Comment by neighborino** [comment]
I'm also having the same issue as @kimown. I followed the instructions above but Atom still opens.

**Comment by lagofvc** [comment]
Same as @neighborino  and @kimown.  The system level icon remains and the file extensions continue pointing to atom and still opens.

**Comment by ashishgkwd** [comment]
Small update, found the `Atom.app` in my home folder on macOS Sierra.
Correct location for deleting is:

`rm -rf ~/Applications/Atom.app`

This finally uninstalled Atom completely :relaxed:

**Comment by jkibele** [comment]
I was having a similar problem. I followed all of the instructions here, but still couldn't get rid of version 1.16. I finally figured out that I had atom in my `Downloads` folder, and that OS X was "helpfully" finding it for me and running it. ...but updating it wouldn't work from there. ...and it also seems that much of my search functionality was set to ignore the `Downloads` folder so it took me forever to figure out what was going on. It's a dumb problem, and I'm embarrassed about it, but it might help somebody out.
